### PR TITLE
[R] Document the sample API doc analyzer

### PR DESCRIPTION
**Pull request by Hixie**

### CODE REVIEW

**Comment by Hixie** [comment]
cc @filiph

**Comment by Hixie** [comment]
cc @cbracken

File: dev/bots/analyze-sample-code.dart [hunk 1]
```diff
@@ -2,6 +2,42 @@
...
+//
+// In addition, you can declare code that should be included in the analysis but
+// not shown in the API docs by adding a comment "// Examples can assume:" to
+// the file (usually at the top of the file, after the imports), following by
+// one or more commented-out lines of code. That code is included verbatim in
+// the analysis.
+//
+// All the sample code of every file is analyzed together. This means you can't
+// have two pieces of sample code that define the same example class, for
+// instance.
```
Comment (cbracken) [inline dev/bots/analyze-sample-code.dart]: uber-nit: I'd just drop the ', for instance' here.

**Review by cbracken — APPROVED** [review]

**Comment by cbracken** [comment]
lgtm

File: dev/bots/analyze-sample-code.dart (hunk 1 above)
Comment (Hixie) [reply dev/bots/analyze-sample-code.dart]: Roger. Will fix in subsequent PR.

Comment (Hixie) [reply dev/bots/analyze-sample-code.dart]: Actually I have to fix merge conflicts anyway, so I'll fix that now.

**Comment by Hixie** [comment]
Landing without flutter-build info because it doesn't seem to be working right now, but tree is green, or as green as we can guess it is with AppVeyor and Travis on break.

**Pull request merged**